devtools-ci-index.yaml

```diff
@@ -89,6 +89,7 @@
         - tsmaeder
         - vikram-raj
         - xcoulon
+        - dfestal
```

Comment:
is it your github user name?

Reply:
oh, no sorry, it's my RedHat one. My GitHub name is `davidfestal`. Let me change.

Reply:
fixed